api/src/main/java/io/minio/PostPolicy.java

```diff
@@ -167,14 +167,6 @@ public String bucketName() {
   }
 
   /**
-   * Returns form data of this post policy.
-   */
-  public Map<String,String> formData(String accessKey, String secretKey)
```

Comment:
Reason of `public formData()` is to access it in `class MinioClient`.  Majority of use cases do not touch/use this method unless some special use case like @MastaP in https://github.com/minio/minio-java/pull/510.

As you use `public Map<String,String> formData(String accessKey, String secretKey, String region)`, you could mark `public Map<String,String> formData(String accessKey, String secretKey)` as deprecated and we will remove it in next major update.